proto/viam/app/v1/app.proto

```diff
@@ -258,8 +261,10 @@ message GetOrganizationResponse {
 
 message UpdateOrganizationRequest {
   string organization_id = 1;
-  string name = 2;
+  optional string name = 2;
```

Comment:
It looks like the UpdateOrganizationRequest was previously only used to change an org's name; any objections to making this change now that we're using this update request for other fields? @michaellee1019

Reply:
If this change is okay I can go ahead and make the corresponding app changes as well.